src/display/font_loader.js

```diff
@@ -270,7 +270,8 @@ var FontLoader = {
       }
       data = spliceString(data, CFF_CHECKSUM_OFFSET, 4, string32(checksum));
 
-      var url = 'url(data:font/opentype;base64,' + btoa(data) + ');';
+      var url = 'url(data:font/opentype;base64,' +
+                  btoa(unescape(encodeURIComponent(data))) + ');';
```

Comment:
Nit: please adjust the indentation such the second line is aligned with the first line, i.e.
```js
var url = 'url(data:font/opentype;base64,' +
          btoa(unescape(encodeURIComponent(data))) + ');';
```

After you've fixed this, please [squash the commits](https://github.com/mozilla/pdf.js/wiki/Squashing-Commits).